nextflow_schema.json

```
@@ -10,13 +10,13 @@
             "type": "object",
             "fa_icon": "fas fa-terminal",
             "description": "Define where the pipeline should find input data and save output data.",
-            "required": ["input", "databases", "outdir"],
+            "required": ["outdir", "databases", "input"],
```

```suggestion
            "required": [ "input", "outdir", "databases"],
```

That way we reduce the diff